tests/unit/h_vialib/client_test.py

```diff
@@ -24,15 +24,17 @@ class TestViaClient:
         "via.client.requestConfigFromFrame.origin": "http://random.localhost",
         "via.client.requestConfigFromFrame.ancestorLevel": "2",
         "via.external_link_mode": "new-tab",
+        "via.sec": Any.string(),
     }
     VIA_URL = "http://via.localhost"
     ORIGIN_URL = "http://random.localhost"
 
-    def test_client_default_options(self, client):
+    def test_client_default_options(self):
+        client = ViaClient(self.VIA_URL, self.ORIGIN_URL, "SECRET")
+
         proxied_url = "http://example.com"
 
         final_url = client.url_for(ViaDoc(proxied_url))
-
         assert final_url == Any.url.matching(
```

Comment:
In general we try to separate the setup, doing and assertion phases of the test, which will have been why this separation was here before.